apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v2/metadata.json

```diff
@@ -150,12 +162,14 @@
       }, {
         "code" : "search-type"
       } ],
+      "searchInclude" : [ "*" ],
+      "searchRevInclude" : [ "Claim:mbi", "ClaimResponse:mbi", "Coverage:beneficiary", "ExplanationOfBenefit:patient" ],
       "searchParam" : [ {
-        "name" : "_has:Coverage",
+        "name" : "_has:Coverage.extension",
```

Comment:
This looks strange at a glance; whats with the extension thing here?

Reply:
These are search parameters. Prior to this version HAPI was printing them out without that last part so it looked like there were two "_has:Coverage" parameters but looks like they fixed that and now we see the complete name. See here for their definitions in the java class: https://github.com/CMSgov/beneficiary-fhir-data/blob/trotter/upgrade-hapi/apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/R4PatientResourceProvider.java#L269